packages/Icon/helper/template.js

```diff
@@ -0,0 +1,12 @@
+module.exports = ({ template }, opts, { imports, interfaces, componentName, jsx, exports }) => {
+  const plugins = ["jsx"];
+  const typeScriptTpl = template.smart({ plugins });
+  return typeScriptTpl.ast`
+  ${imports}
+  ${interfaces}
+  function ${componentName}({title=null, ...props}) {
```

Comment:
I guess this is trigger by typescript and modified the `ast` while executing the code? or how does this works?

Reply:
This triggered by svgr plugin https://github.com/gregberge/svgr/blob/master/packages/babel-plugin-transform-svg-component/src/index.js
You can find the default template on that page as well